### ISSUE TITLE
Topbar - Toolbar primitive images are greyed out and don't work. Adding images works elsewhere.

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#4844]
![image](https://github.com/user-attachments/assets/a7b0b6eb-e7e4-401d-995e-bea2e996caa8)

Seems they are the "legacy" operators and may need an update in both topbars.

**Comment by Blenux** [comment]
They were only allowed to be used in the view3d area it seems.

![image](https://github.com/user-attachments/assets/90954649-6bfe-457a-9918-c4448fa7b429)

remove that and they aren't greyed out (and work)
![image](https://github.com/user-attachments/assets/5e3f6071-ad84-484a-ad31-34753adbdd6b)

but the older toolbar does need updates to it.

![image](https://github.com/user-attachments/assets/2c1fcb24-5e3a-430d-8396-accbdd53b1df)

**Comment by Blenux** [comment]
hmm the image background one crashes the app when using in the toolbars.

**Comment by ReinerBforartists** [comment]
Yeah, these operators are meant to be used from within the 3D view. It was and is one of the limits, unfortunately. And the reason why not all tools are available from the toolbar.

**Comment by Draise14** [comment]
I guess it's fine if removed, these are added by drag and drop anyway..

**Comment by ReinerBforartists** [comment]
Yep, overboard with it !

**Comment by Blenux** [comment]
ok cool will do today
